src/deroff.rs

```diff
@@ -273,6 +273,51 @@ impl Deroffer {
             _ => return None,
         })
     }
+    //     def skip_char(self, amt=1):
+    //         self.s = self.s[amt:]
+    fn skip_char<'a>(&self, s: &'a str, amount: Option<usize>) -> &'a str {
+        let amount = amount.unwrap_or(1);
+        s.get(amount..).unwrap_or("")
+    }
+
+    //     def skip_leading_whitespace(self):
+    //         self.s = self.s.lstrip()
+    fn skip_leading_whitespace<'a>(&self, s: &'a str) -> &'a str {
+        s.trim_start()
+    }
+    //     def str_at(self, idx):
```

Comment:
nit! newline before comment

Reply:
I'd hoped `cargo fmt` would do this, I don't think too much about formatting anymore :)